source/eitri/docker_wrapper.py

```diff
@@ -63,12 +90,16 @@ def old_school_cnx_string(self):
 
 class PostgresDocker(object):
     """
-    launch a temporary docker with a postgresql-postgis db
+    Class to launch a temporary docker with a PostgreSQL/PostGIS database.
     """
 
     def __init__(self):
-        log = logging.getLogger(__name__)
-        base_url = 'unix://var/run/docker.sock'
+        # type: () -> None
+        """
+        Constructor of PostgresDocker.
+        """
+        log = logging.getLogger(__name__)  # type: logging.Logger
+        base_url = 'unix://var/run/docker.sock'  # type: str
```

Comment:
Is it useful in this case? Clearly, it's a string!

Reply:
If you want to use mypy you have to tell him what is what ;)
This comment allow him to know base_url is a string and then, when you use it in a function (that also have to comment for mypy) it will detect an error type.

Reply:
I know that. The way I see it, mypy helps us to have the type needed in a variable passed to a function or returned by a function. Here, the variable is set directly, it can't be anything else, so the gain offered by mypy is quite limited IMO. 
We'll call a referee :checkered_flag:

Reply:
I would agree with @mbounabi, there is very little benefits documenting an initialised variable like this one, which clearly is a `string`.

What mypy error do you actually have? would `base_url = str('unix://var/run/docker.sock')` help?

Reply:
My bad, it's not needed for strings.